### DISCUSSION TITLE
How can I stretch (fill) the texts?

### DISCUSSION THREAD

**Discussion by NVHT** [moudey/Shell#158] [Q&A]
![image](https://user-images.githubusercontent.com/8341213/217809773-1d88041c-96f8-4fd4-b6c5-3adb550461d0.png)

How can I make the text stretch and fill the space?

**Comment by moudey** [comment]
> How can I make the text stretch and fill the space?

Do you want to increase the width?

To reduce size
```
theme
{
    item.padding=[2,auto]
    item.margin=[0, auto]
    
    border.padding=[4,6]
}
```

**Reply by NVHT** [reply]
![image](https://user-images.githubusercontent.com/8341213/217827642-3b9a4958-ec4d-4fd2-852c-3b8af7968707.png)
I want to increase the width. I want to move the two elements to the edge

![image](https://user-images.githubusercontent.com/8341213/217828103-5a81b00e-cb83-4c38-92a5-16d8b40b620d.png)
Basically like the original, but themed.

**Reply by moudey** [reply]
I will add a `width` option to `theme.layout` to set the minimum width

**Comment by moudey** [comment]
Try this [build](https://github.com/moudey/Shell/files/10698443/shell-test.zip)
You can set the minimum and maximum width
```
theme
{
    layout.width=300 // minimum 
    layout.width=[300, 500] // [min, max]
}
```

**Reply by NVHT** [reply]
Works great now. Thanks.
